### PR TITLE
Added ability to pass in custom Git URLs as project type

### PR DESCRIPTION
**Pull request by aladage**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@deg-skeletor/orko",
-    "version": "1.0.2",
+    "version": "1.0.3",
```
Comment (heaper) [inline package.json]: Would this be a non-breaking change/enhancement rather than a bug fix?

Comment (aladage) [reply package.json]: Ah, good call. Updated to 1.2.0

**Comment by heaper** [comment]
Hey! Run your tests, cowboy!

**Comment by heaper** [comment]
Looks good. Nice one single test you have there.

**Pull request merged**